trulens_eval/examples/expositional/models/arctic_quickstart.ipynb

```diff
@@ -4,14 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    from trulens_eval import Cortex
</pre><p>can you import from <code>trulens_eval.feedback.provider</code>instead of toplevel?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>